KFSPTS 10681 WebServiceCredential Maintenance Document

ajd299 opened the pull request.

jhulslander commented on src/main/resources/edu/cornell/kfs/sys/businessobject/datadictionary/WebServiceCredential.xml:
```diff
@@ -0,0 +1,151 @@
+        </property>
+    </bean>
+
+    <!-- Attribute Definitions -->
```
these comments could be removed

jhulslander commented on src/main/java/edu/cornell/kfs/sys/document/authorization/WebServiceCredentialAuthorizer.java:
```diff
@@ -0,0 +1,31 @@
+        }
+
+        WebServiceCredential webServiceCredential = (WebServiceCredential) dataObject;
+        ParameterEvaluator parameterEvaluator = SpringContext.getBean(ParameterEvaluatorService.class).getParameterEvaluator(
```
I would move ParameterEvaluatorService into a getter and setter, and use spring conect in the getter in the local variable for ParameterEvaluatorService isn't set.

ajd299 commented:
I've removed the comments from the xml files and implemented member variable and public getter/setter functions. Please review the changes and let me know if this is the best practice.
